contrib/hack/gen-deploy.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE}")
+REPO_ROOT="${SCRIPT_ROOT}/../.."
+
+gen() {
+	OUTPUT=$1
+    VALUES_PATH=""
+	TMP_OUTPUT=$(mktemp)
+	mkdir -p "$(dirname ${OUTPUT})"
+    if [[ ! -z "$2" ]]; then
+        VALUES=$2
+        VALUES_PATH="--values=${SCRIPT_ROOT}/deploy/values/${VALUES}.yaml"
+    fi
+    helm template \
+        "${REPO_ROOT}/charts/azure-k8s-metrics-adapter/" \
+        --namespace "custom-metrics" \
+        --name "azure-k8s-metrics-adapter" \
+        --set "fullnameOverride=azure-k8s-metrics-adapter" \
+		--set "createNamespaceResource=true" > "${TMP_OUTPUT}" \
```

Comment:
this is the `createNamespacedResource` again.  I don't think it is used anywhere else....

Reply:
That value will only be used to add the Namespace API object definition during the generation of the static deployment file, see https://github.com/Azure/azure-k8s-metrics-adapter/pull/14/files#diff-a90ec40e23644a6d5db9642bc738242aR1

Reply:
Ah missed that.  Thank for pointing it out!

Reply:
could you also add note to this file that it was modified from the cert-manager and link to their license and original file?